docs/architecture/grpc-for-wcf-developers/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: ASP.NET Core gRPC for WCF Developers
+description: A guide for building cloud native applications and microservices using gRPC, for WCF developers seeking to modernize their solutions.
+author: markrendle
+ms.date: 28/08/2019
```

Comment:
format is mm/dd/yyyy 

```suggestion
ms.date: 08/28/2019
```

Reply:
Only in America... 😉 

(Fixed it anyway)